PlayCover/Utils/SaveGenshinUserData.swift

```diff
@@ -8,17 +8,21 @@
 import Foundation
 import SwiftUI
 
-func storeUserData( folderName: String, accountRegion: String ) {
+func storeUserData( folderName: String, accountRegion: String, app: PlayApp ) {
+        let BUNDLE_ID = app.info.bundleIdentifier == "com.miHoYo.GenshinImpact" ? "com.miHoYo.GenshinImpact"
```

Comment:
Line can be simplified to remove the conditional. `BUNDLE_ID` should also be changed for `bundleID`